widget/embedded/src/components/Quote/Quote.types.ts

```diff
@@ -9,10 +15,18 @@ export type QuoteProps = {
   warning: QuoteWarning | null;
   recommended: boolean;
```

Comment:
I feel like using the  recommended property doesn't really make sense right now. We use it when a quote is selected, and it has nothing to do with the recommended quote.

Reply:
fixed